core/src/main/java/hudson/tasks/Fingerprinter.java

```diff
@@ -92,13 +93,40 @@
      */
     private final String targets;
 
+    /**
+     * Default null 'excludes' pattern as in Ant.
+     */
+    private String excludes = null;
+
+    /**
+     * Default ant exclusion
+     */
+    private Boolean defaultExcludes = true;
+
+    /**
+     * Indicate whether include and exclude patterns should be considered as case sensitive
+     */
+    private Boolean caseSensitive = true;
+
     @Deprecated
     Boolean recordBuildArtifacts;
 
     @DataBoundConstructor public Fingerprinter(String targets) {
         this.targets = targets;
     }
 
+    @DataBoundSetter public void setExcludes(String excludes) {
+        this.excludes = excludes;
+    }
+
+    @DataBoundSetter public void setDefaultExcludes(boolean defaultExcludes) {
```

Comment:
```suggestion
    @DataBoundSetter public void setDefaultExcludes(Boolean defaultExcludes) {
```

I'm only considering this one for consistency with my suggestions for the getters, though it may not be necessary.